resources/languages/cpp/AnsiCodes.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright 1996-2020 Cyberbotics Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#define WB_ALLOW_MIXING_C_AND_CPP_API
+#include <webots/utils/ansi_codes.h>
+#include <webots/utils/AnsiCodes.hpp>
+
+using namespace webots;
+
+const std::string AnsiCodes::RESET = ANSI_RESET;
```

Comment:
Can't all of those be inside the AnsiCode.hpp ?

Reply:
I don't think so. If you have a better idea, please test it and suggest it here.

Reply:
I tried a couple of things without success, It's a pity it's not possible to simply have defines translated to constants automatically by swig...

Reply:
Yes, swig has some bad limitations about this.